.php_cs.dist

```diff
@@ -23,7 +23,9 @@ $finder = PhpCsFixer\Finder::create()
     ])
     ->append([
         __FILE__,
+        __DIR__.'/index.php',
```

Comment:
Die Soll nicht gefixt werden, wegen php-Kompatibilität nach unten, um zum PHP-Check in der boot zu gelangen

Reply:
Ok